internal/configuration/validator/server.go

```diff
@@ -2,15 +2,28 @@ package validator
 
 import (
 	"fmt"
+	"path"
+	"strings"
 
 	"github.com/authelia/authelia/internal/configuration/schema"
+	"github.com/authelia/authelia/internal/utils"
 )
 
 var defaultReadBufferSize = 4096
 var defaultWriteBufferSize = 4096
 
 // ValidateServer checks a server configuration is correct.
 func ValidateServer(configuration *schema.ServerConfiguration, validator *schema.StructValidator) {
+	switch {
+	case strings.Contains(configuration.Path, "/"):
+		validator.Push(fmt.Errorf("server path must not contain any forward slashes"))
```

Comment:
why not support this?

Reply:
I believe it had to do with how the path is cleaned, though James can probably clarify.

It's also to avoid any confusion for users attempting to publish multiple level routes, ie `/auth/authelia`. We only want to support base_url that's a single level deep.

Reply:
Ok, good enough for now and until someone ask for more (it will likely happen in my opinion).

Reply:
I also wanted to keep the input as controlled as possible so we know what we're working with. Some things expect a leading slash, some do not.